creator/data_reviews/tasks.py

```diff
@@ -0,0 +1,48 @@
+
+
+def start_review():
```

Comment:
I'm not sure about structuring things this way.
Perhaps we instead have tasks for each lengthy action that needs to happen, like emails, and invoke them from the transition or the signal. That way, we don't have all of our tasks executing in one task invocation.

Reply:
Yea I wasn't sure about this either. I can remove the task placeholders